### PR TITLE
feat(frontend): redesign navbar

### PR DESCRIPTION
**Pull request by zaxutic**
this is part of a pr train: relies on #196, which relies on #200 (change this pr's target to main and rebase after merging those)

implements new navbar design. includes a link to a theoretical About page which doesn't exist yet

### CODE REVIEW

File: frontend/src/utils/index.js [hunk 1]
```diff
@@ -10,9 +10,7 @@ export function isAdmin() {
   return true;
 }
 
-export function isLoggedIn() {
-  return true;
-}
+export const isLoggedIn = () => localStorage.hasOwnProperty("AUTH_TOKEN");
```
Comment (lhjt) [inline frontend/src/utils/index.js:13]: Do you plan to move this elsewhere in the future?

Comment (zaxutic) [reply frontend/src/utils/index.js:13]: hmm would something like the `api` module be a better place for this?

Comment (lhjt) [reply frontend/src/utils/index.js:13]: Oh no, I meant more that you're storing the auth token in local storage (as in do you plan to use a more secure method of storing auth state at some point)

Comment (zaxutic) [reply frontend/src/utils/index.js:13]: Oh right, yeah we're planning to move it to cookies instead

Comment (lhjt) [reply frontend/src/utils/index.js:13]: (This is irrelevant to this PR, and hence it's a comment rather than a change request; sounds good)

**Review by lhjt — APPROVED** [review]

**Pull request merged**